test/e2e/cross_os/test_worker_status.py

```diff
@@ -0,0 +1,72 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+"""
+This test module contains tests that verify the Worker agent's behavior by starting/stopping the Worker,
+and making sure that the status of the Worker is that of what we expect.
+"""
+import pytest
+from typing import Any, Dict
+from deadline_worker_agent.api_models import (
+    WorkerStatus,
+)
+import backoff
+from deadline_test_fixtures import DeadlineClient, EC2InstanceWorker
+from utils import get_operating_system_name
+import pytest
+
+
+@pytest.mark.parametrize("operating_system", [get_operating_system_name()], indirect=True)
```

Comment:
are we planning on adding more parameters later on? Also, I'm not familiar with using parameters at test class level. What's the purpose of this? Could it be a fixture instead?